integration-tests/narayana-lra/src/test/java/org/acme/quickstart/lra/AppTestProfile.java

```diff
@@ -0,0 +1,14 @@
+package org.acme.quickstart.lra;
+
+import java.util.List;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+
+// we use a QuarkusTestProfile to ensure that QuarkusIntegrationTest works properly with them
+public class AppTestProfile implements QuarkusTestProfile {
```

Comment:
Shouldn't that be in `integration-tests/main`? What if `narayana-lra` is moved out of the Quarkus main repo at some point? Wouldn't we then loose coverage of this feature?

Reply:
Indeed, but we can cross that bridge if we get to it :)

Reply:
😄 

My point is: Will _anyone_ remember to look that closely? Anyway, I just find it confusing that coverage of some test framework aspects is scattered over the repo which makes it hard to validate changes without running full CI.

Reply:
I will remember :)